### PR TITLE
DOC: remove grammar duplication in groupby docs

### PR DESCRIPTION
**Pull request by adamjstewart**
"can be could be" -> "could be"

### CODE REVIEW

**Review by WillAyd — APPROVED** [review]

**Comment by gfyoung** [comment]
Marking this for `0.24.0`, as this is a relatively trivial PR, and we can pick this up and bring to the finish line (I only have one comment, otherwise, LGTM) if it goes stale.

**Review by jorisvandenbossche — APPROVED** [review]
Done

**Comment by jorisvandenbossche** [comment]
@adamjstewart Thanks!

**Pull request merged**
